docs/config-example.md

```diff
@@ -28,6 +28,10 @@ spec:
       - ntp.aliyun.com
       - node1 # Set the node name in `hosts` as ntp server if no public ntp servers access.
     timezone: "Asia/Shanghai"
+    rpms: # Offline install custom rpm packages
```

Comment:
Suggest comment:

Specify additional packages to be installed. The ISO file which is contained in the artifact is required.